bin/graftM

```diff
@@ -71,32 +71,35 @@ def phelp():
     print """
                                        GraftM  %s
 
-             A suite of tools for the rapid analysis of metagenome sequence datasets.
-
                                 Joel Boyd, Ben Woodcroft
 
+    GraftM is a tool for rapid, phylogenetically informed gene-centric analysis of sequence 
+    data. The GraftM pipeline can be split into two steps. The first is the identification of 
+    reads of interest using Hidden Markov Models (HMMs), and the second is the placement 
+    of these reads into phylogenetic trees. The classification of sequences by GraftM is inferred 
+    by the placement of each sequence relative to annotated references in the tree. 
```

Comment:
How about this slightly modified version?

GraftM is a tool for rapid, phylogenetically informed
gene-centric analysis of sequence data. The main 'graft' pipeline
identifies short read sequences with homology to genes of
interest (16S rRNA or protein coding) using Hidden Markov
Models (HMMs), which are then placed into a phylogenetic
tree. The classification of sequences is inferred by their
placement relative to annotated reference sequences in the
tree. For protein coding genes, a 'best BLAST hit' style analysis
can also be used.

A manuscript describing and benchmarking the tool is in preparation:

Boyd, J., Woodcroft B., Tyson, G. "GraftM: A tool for scalable,
phylogenetically informed classification of"genes within
metagenomes" (in preparation).